Updates to Dutch 8 dot table

leonardder opened the pull request:
The Dutch 8 dot braille working group of the Dutch braille authority have changed their braille table draft. This draft will likely be equal to the final version.
I'd like to have a review from @dkager as well.

Closes #954

bertfrees commented:
Thanks Leonard.

If this will be the final version I think a reference in the table and test to the specification would be appropriate. `#+version` and `#-display-name` also need update.

bertfrees commented:
@leonardder Planning more changes this week?

bertfrees commented:
@dkager Do you have time to look at these changes to the 8-dot table before the end of the week?

dkager commented:
Yes. Apologies for the delay.

Van: Bert Frees <notifications@github.com>
Verzonden: maandag 25 mei 2020 15:24
Aan: liblouis/liblouis <liblouis@noreply.github.com>
CC: Davy Kager <DavyKager@dedicon.nl>; Mention <mention@noreply.github.com>
Onderwerp: Re: [liblouis/liblouis] Updates to Dutch 8 dot table (#956)


@dkager<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdkager&data=01%7C01%7Cdavykager%40dedicon.nl%7Ca832a452a32e4e6cf76208d800aef2ea%7Ccc5d43a1454b4d239af522c5220bc61b%7C0&sdata=7xrHjky2hAHF%2FVNGIPobOl%2FrwOR9JGRHZwKzUnpVzqw%3D&reserved=0> Do you have time to look at these changes to the 8-dot table before the end of the week?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fliblouis%2Fliblouis%2Fpull%2F956%23issuecomment-633571050&data=01%7C01%7Cdavykager%40dedicon.nl%7Ca832a452a32e4e6cf76208d800aef2ea%7Ccc5d43a1454b4d239af522c5220bc61b%7C0&sdata=5%2B2OpbRf2ItzU%2BJamYKFmgYAEOFea07%2FSpedO8pjVdk%3D&reserved=0>, or unsubscribe<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAAXXF4MFZDDTYFIQ3GRANVTRTJWQTANCNFSM4NBLT7LQ&data=01%7C01%7Cdavykager%40dedicon.nl%7Ca832a452a32e4e6cf76208d800aef2ea%7Ccc5d43a1454b4d239af522c5220bc61b%7C0&sdata=pQJkch6wlGDurSgnyFzedsuUMNPs1AADzzZrfIv6WSk%3D&reserved=0>.

DISCLAIMER:
De informatie verzonden met dit e-mail bericht is uitsluitend bestemd voor de geadresseerde. Indien u niet de beoogde geadresseerde bent, verzoeken wij u vriendelijk dit aan de afzender te melden (of via: info@dedicon.nl<mailto:info@dedicon.nl>) en het origineel en eventuele kopieën te verwijderen.

The information sent in this e-mail is solely intended for the individual or company to whom it is addressed. If you received this message in error, please notify the sender immediately (or mail to info@dedicon.nl<mailto:info@dedicon.nl>) and delete the original message and possible copies.

dkager commented:
Looks good!

leonardder commented:
@bertfrees the Dutch braille authority agreed on making this the final 2020.1 version. We're currently working on a public specification document, but I'm afraid it won't be ready before the release.
Do you have any additional requests to change things?

bertfrees commented:
Thanks!

> Do you have any additional requests to change things?

An update of nl-comp8_harness.yaml would be nice.

leonardder commented:
I updated nl-comp8_harness.yaml and added nl-comp8-for_harness.yaml for greek braille, which is added for forward translation only.
I'm afraid I don't have any pending time to add more tests just yet. May be for the next release.

bertfrees commented:
Thanks! If you change that "_may be_ for the next release" to "_certainly_ for the next release", it's fine :-)

leonardder commented:
That's certainly my intention.

bertfrees commented:
https://github.com/liblouis/liblouis/issues/964

bertfrees commented:
@leonardder Your test actually revealed a bug in the table: the word-final position variant of the sigma is missed. Do you want to fix that before I merge?

bertfrees commented:
Or I can do it if you confirm the test is correct and the table is wrong.

leonardder commented:
I believe the final sigma was left out from the spec on purpose, so the test was wrong and I have removed it. I will discuss this with the Responsible people. @dkager do you know why this was done?

dkager commented:
See my post on the mailing list. In short: we only included those characters that are relevant to maths. My knowledge of Greek and maths is not good enough to know if final sigma belongs to the maths world as well.

Van: Leonard de Ruijter <notifications@github.com>
Verzonden: maandag 1 juni 2020 18:40
Aan: liblouis/liblouis <liblouis@noreply.github.com>
CC: Davy Kager <DavyKager@dedicon.nl>; Mention <mention@noreply.github.com>
Onderwerp: Re: [liblouis/liblouis] Updates to Dutch 8 dot table (#956)


I believe the final sigma was left out from the spec on purpose, so the test was wrong and I have removed it. I will discuss this with the Responsible people. @dkager<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdkager&data=01%7C01%7Cdavykager%40dedicon.nl%7C503f9f7d3b5d477de50208d8064a7378%7Ccc5d43a1454b4d239af522c5220bc61b%7C0&sdata=0ZXoo8%2BEfnWJ1DILcQbCObEsI0w%2BjXyTqBnz9mefxLg%3D&reserved=0> do you know why this was done?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fliblouis%2Fliblouis%2Fpull%2F956%23issuecomment-636976585&data=01%7C01%7Cdavykager%40dedicon.nl%7C503f9f7d3b5d477de50208d8064a7378%7Ccc5d43a1454b4d239af522c5220bc61b%7C0&sdata=ueMQOBFlDGqh1nKzDweI4l6qyKjdPQdAM6JUChJ9sl0%3D&reserved=0>, or unsubscribe<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAAXXF4JTRDRXABVOPLG5JMTRUPKWTANCNFSM4NBLT7LQ&data=01%7C01%7Cdavykager%40dedicon.nl%7C503f9f7d3b5d477de50208d8064a7378%7Ccc5d43a1454b4d239af522c5220bc61b%7C0&sdata=PyB%2BqMg3bZ%2BK4GXEwuWG692ELpuBfWcAuERmwOVsIhg%3D&reserved=0>.

DISCLAIMER:
De informatie verzonden met dit e-mail bericht is uitsluitend bestemd voor de geadresseerde. Indien u niet de beoogde geadresseerde bent, verzoeken wij u vriendelijk dit aan de afzender te melden (of via: info@dedicon.nl<mailto:info@dedicon.nl>) en het origineel en eventuele kopieën te verwijderen.

The information sent in this e-mail is solely intended for the individual or company to whom it is addressed. If you received this message in error, please notify the sender immediately (or mail to info@dedicon.nl<mailto:info@dedicon.nl>) and delete the original message and possible copies.

bertfrees commented:
I don't remember if I ever stumbled upon that variant of the sigma in my math classes. Probably not.
